### PR TITLE
Feature - Rx Search

### PR DESCRIPTION
**Pull request by at-sunguyen**
- Rx search

### CODE REVIEW

File: UnittestExample/View/Base/TabbarViewController.swift [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+//  UnittestExample
+//
+//  Created by Su Nguyen T. on 9/18/18.
+//  Copyright © 2018 IOS Testing. All rights reserved.
+//
+
+import UIKit
+
+final class TabbarViewController: UITabBarController {
+
```
Comment (at-lamle) [inline UnittestExample/View/Base/TabbarViewController.swift]: WARNING - Lines should not have trailing whitespace.

**Comment by at-lamle** [comment]
Total linter violations in pull request: 1
Serious: 0
Warnings: 1

**Comment by at-lamle** [comment]
:+1: There are no linter violations.

**Comment by at-sunguyen** [comment]
@at-haiphung Please review this pr, thanks!

**Pull request merged**